### DISCUSSION TITLE
Why does `tsc` with `"vitest"` version 2 require `"@types/node"` to run without errors?

### DISCUSSION THREAD

**Discussion by hirehamir** [vitest-dev/vitest#6117] [Q&A]
`"vitest"` version 1 doesn't seem to require `"@types/node"`. I was just curious why.

**Comment by hirehamir** [comment]
You can see that [`tsc --noEmit` fails](https://github.com/hamirmahal/vitest-debug/actions/runs/9915513422/job/27396470395#logs) after [updating from `"vitest": "^1.6.0"` to `"vitest": "^2.0.2"`](https://github.com/hamirmahal/vitest-debug/commit/dff58217f125fc8b71776ab73c9bdd9fa37b5811), whereas [it was passing before](https://github.com/hamirmahal/vitest-debug/commits/tsc-fails-without-node-types-for-vitest-v2/).

**Reply by sheremet-va** [reply]
This is a bug

**Reply by sheremet-va** [reply]
The main `vitest` entry point types are a mess, but changing it will be a breaking change. Please open a separate issue with your reproduction so it's easier to track 😄

**Reply by hirehamir** [reply]
https://github.com/vitest-dev/vitest/issues/6141
